sphinx/ext/autosummary/generate.py

```diff
@@ -398,9 +427,12 @@ def _skip_member(app: Sphinx, obj: Any, name: str, objtype: str) -> bool:
         return False
 
 
-def _get_class_members(obj: Any) -> dict[str, Any]:
+def _get_class_members(obj: Any, return_object: bool = True) -> dict[str, Any]:
```

Comment:
I would actually flip the boolean and use `return_types=False` by default. What do you think?